pkgs/by-name/as/asdf-vm/package.nix

```diff
@@ -68,30 +68,31 @@ stdenv.mkDerivation rec {
     git
   ];
 
+  dontBuild = true;
+
   installPhase = ''
+    runHook preInstall
+
     mkdir -p $out/share/asdf-vm
     cp -r . $out/share/asdf-vm
 
     mkdir -p $out/etc/profile.d
     substitute ${asdfPrepareFile} $out/etc/profile.d/asdf-prepare.sh \
-      --replace "@asdfDir@" "$out/share/asdf-vm"
+      --replace-fail "@asdfDir@" "$out/share/asdf-vm"
 
     mkdir -p $out/bin
     makeWrapper $out/share/asdf-vm/bin/asdf $out/bin/asdf \
       --set ASDF_DIR $out/share/asdf-vm
 
-    installShellCompletion --cmd asdf \
-      --zsh completions/_asdf \
-      --fish completions/asdf.fish \
-      --bash completions/asdf.bash
+    runHook postInstall
```

Comment:
The completions seems to be in `$out/share/asdf-vm/internal/completions/`, please make sure they're installed properly